deploy/olm-catalog/ibm-block-csi-operator/1.6.0/manifests/ibmblockcsis.csi.ibm.com.crd.yaml

```diff
@@ -16,280 +16,391 @@ spec:
     listKind: IBMBlockCSIList
     plural: ibmblockcsis
     shortNames:
-    - ibc
+      - ibc
```

Comment:
is this indentation special for certified?

Reply:
This is an auto-generated code.
How can I know if this is special for OLM?

Reply:
I asked it because I don't see this change in the non-OLM crd yaml.
where is the logic that syncs our crd duplications?

Reply:
The `operator-sdk` generates the YAML files (by using `operator-sdk generate crds`).
I didn't change the indentations afterward.

Reply:
great, so make sure they are all in sync

Reply:
Done

Reply:
still doesn't look in sync to me.
e.g. indentations between `csi.ibm.com_ibmblockcsis_crd.yaml` and `ibm-block-csi-operator.yaml` - are you sure all is aligned?

Reply:
As I said, I didn't change the indentations for the auto-generated file. We shouldn't edit it manually. CSI-1298 will address this file.

Reply:
we shouldn't leave it unaligned, and since we still don't have auto sync yet, we should keep it aligned, even if it requires manual sync

Reply:
Why not? It worked this way so far. Why it's important to align it?

Reply:
we don't want to maintain more than 1 version for each of our duplicated yamls

Reply:
Done